src/main/java/fr/mrmicky/fastboard/FastReflection.java

```diff
@@ -42,8 +42,9 @@ public final class FastReflection {
     private static final String NM_PACKAGE = "net.minecraft";
     public static final String OBC_PACKAGE = "org.bukkit.craftbukkit";
     public static final String NMS_PACKAGE = NM_PACKAGE + ".server";
+    private static final String OBC_PREFIX = Bukkit.getServer().getClass().getPackage().getName();
```

Comment:
Great, just maybe wouldn't something like this be more convenient?

```suggestion
    private static final String OBC_PACKAGE = Bukkit.getServer().getClass().getPackage().getName();
    private static final String NMS_PACKAGE = OBC_PACKAGE.replace("org.bukkit.craftbukkit", "net.minecraft.server");
```

And then remove `'.' . VERSION` from `nmsClassName`/`obcClassName` methods

Reply:
I was reluctant to make breaking changes to `public` fields, but I went ahead and implemented your suggestion.